MapboxNavigation/NavigationViewController.swift

```diff
@@ -154,6 +154,19 @@ public protocol NavigationViewControllerDelegate {
      Returns the center point of the user course view in screen coordinates relative to the map view.
      */
     @objc optional func navigationViewController(_ navigationViewController: NavigationViewController, mapViewUserAnchorPoint mapView: NavigationMapView) -> CGPoint
+    
+    /**
+     Returns whether the `BottomBannerView` in the `RouteMapViewController` should show a cancel button.
+     
+     If implemented, this method is called at `RouteMapViewController`'s `viewWillAppear()` time. This is used to initally determine whether the cancel button should be shown when the view first appears. If you need to set the visibility of the cancel button after this point, manipulate the `BottomBannerView.isShowingCancelButton` property directly.
```

Comment:
Is an explicit setup method necessary? This note makes it sound like the developer can accomplish the same effect (from the user’s perspective) by setting `isShowingCancelButton` between the call to `init(for:directions:styles:locationManager:)` and `viewDidAppear()`, such as in `prepare(for:)`.